graphene/core/schema.py

```diff
@@ -77,6 +77,7 @@ def schema(self):
             self,
             query=self.T(self.query),
             mutation=self.T(self.mutation),
+            types=[self.T(_type) for _type in list(self._types_names.values())],
```

Comment:
@ekampf in Python 3 dict.values() returns an iterable that raises an error if the dict is changed in runtime. Casting to a list fix this problem.

See the related failed testing: https://travis-ci.org/graphql-python/graphene/builds/133549359